Formula/nushell.rb

```diff
@@ -1,9 +1,8 @@
 class Nushell < Formula
   desc "Modern shell for the GitHub era"
   homepage "https://www.nushell.sh"
-  url "https://github.com/nushell/nushell/archive/0.2.0.tar.gz"
-  sha256 "5bce8cdb33a6580ff15214322bc66945c0b4d93375056865ad30e0415fece3de"
-  revision 1
+  url "https://github.com/nushell/nushell/archive/0_5_0.tar.gz"
```

Comment:
Maybe good to use this one?
`https://github.com/nushell/nushell/releases/download/0_5_0/nu_0_5_0_macOS.zip`

Reply:
The `nu_0_5_0_macOS.zip` file contains binaries rather than source.  We definitely want to be building from source using the `tar.gz` file in this PR.

Reply:
Even if it didn't contained binaries it seems to be platform specific which is highly discouraged in homebrew-core.